mongofiles/mongofiles_test.go

```diff
@@ -372,63 +394,49 @@ func TestMongoFilesCommands(t *testing.T) {
 			})
 		})
 
-		Convey("Testing the 'get' command with a file that is in GridFS should", func() {
-			mf, err := simpleMongoFilesInstanceWithFilename("get", "testfile1")
+		Convey("Testing the 'get' command with files that are in GridFS should", func() {
+			testFiles := []string{"testfile1", "testfile2", "testfile3"}
```

Comment:
Instead of creating three test files, i.e. of the form `testfile(\d)`, create four -- then, test that `mongofiles ... get ...` operates only on three, but not four, test files.